main.go

```
@@ -1,7 +1,52 @@
+		supabaseClient: client,
+	}
+
+	r := gin.Default()
+
+	r.POST("/register", cfg.handlerRegisterUser)
+	r.POST("/login", cfg.handlerLoginUser)
+	r.GET("/health", func(c *gin.Context) {
+		c.JSON(http.StatusOK, gin.H{"status": "ok"})
+	})
+
+	r.Run(fmt.Sprintf(":%s", PORT))
```

golangci-lint (1.62.2)

Error return value of `r.Run` is not checked

(errcheck)